services/frontend-service/src/ui/components/CommitInfo/CommitInfo.tsx

```diff
@@ -134,5 +134,26 @@ const eventDescription = (event: Event): [JSX.Element, string] => {
                     );
             }
             return [description, de.targetEnvironment];
+        case 'lockPreventedDeploymentEvent':
+            const inner = tp.lockPreventedDeploymentEvent;
+            return [
+                <span>
+                    Application <b>{inner.application}</b> was blocked from deploying due to{' '}
```

Comment:
It's probably useful to indicate the environment here as well.

Reply:
Isn't that duplicating the information that is already in the "environment" column?

Reply:
I don't mind either way for now. We'll unify this later for all lock types.